### PR TITLE
Fix wrong accuracy value in tutorial...

### PR DESCRIPTION
**Pull request by ManishAradwad**
Fixes : https://github.com/tensorflow/tensorflow/issues/35249
The accuracy of model on training data after 10 epochs is 91% while it is mentioned as 88%

### CODE REVIEW

**Comment by ManishAradwad** [comment]
@lakovar @MarkDaoust @yashk2810 
Can you plz see if this PR is fine??

**Comment by lamberta** [comment]
Your formatting makes it difficult to read diffs. Can you please download the ipynb from Colab? Thanks
